logever/src/main/java/ai/smartfac/logever/controller/UserController.java

```diff
@@ -33,7 +33,20 @@ public ResponseEntity<?> getUsers(@PathVariable("username") String username) {
 
     @PostMapping("/register")
     public ResponseEntity<?> registerUser(@RequestBody User user) {
```

Comment:
I think we should use DTOs or separate requests class same as RolePermissionRequest instead of directly using entity class in controller request. We should also add validation on request body.

Reply:
If the request body is diff then it makes sense to use a diff model.
2nd point is required. Validation should be added. going fwd along with some other dev we can cover validations for all requests